src/v/transform/rpc/client.cc

```diff
@@ -330,4 +338,197 @@ client::compute_wasm_binary_ntp_leader() {
     }
     co_return leader;
 }
+
+ss::future<result<model::partition_id>>
+client::find_coordinator(model::transform_offsets_key key) {
+    // todo: lookup in a local cache first.
+    return retry_with_backoff(
+      max_client_retries, [&, this] { return find_coordinator_once(key); });
+}
+
+ss::future<result<model::partition_id>>
+client::find_coordinator_once(model::transform_offsets_key key) {
+    auto ntp = offsets_ntp(coordinator_partition);
```

Comment:
Should we be creating this topic here if it doesn't exist - or is there another place we generally create these internal topics?

Reply:
I left it out assuming we'd do it some place centrally.. during rebase I realized there is a new 'topic_creator' interface for a similar usecase with wasm_binary topic, hooked this one up as well. Thanks for flagging.